project.toml

```diff
@@ -3,4 +3,32 @@ requires = [
     "setuptools>=42",
     "wheel"
 ]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ivy"
+version = "{__version__}"
```

Comment:
We import this value from `ivy/_version.py`. Does that heppen?

Ref: https://github.com/unifyai/ivy/blob/master/setup.py#L53